From 452ab0c5af5f30b7436f19f0fdc837ffb2b6556f Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Mon, 10 Jun 2019 22:03:10 +0200
Subject: [PATCH 29/37] SystemUI: Offset battery circle frame by [-2.0f, 0]

* That makes it so that it's aligned with portrait battery.

Change-Id: I62013cac7df337c4b793ba23c85160c3c23ea897
---
 .../com/android/settingslib/graph/BatteryMeterDrawableBase.java  | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SettingsLib/src/com/android/settingslib/graph/BatteryMeterDrawableBase.java b/packages/SettingsLib/src/com/android/settingslib/graph/BatteryMeterDrawableBase.java
index 32341c9605c..1ac82b16acf 100644
--- a/packages/SettingsLib/src/com/android/settingslib/graph/BatteryMeterDrawableBase.java
+++ b/packages/SettingsLib/src/com/android/settingslib/graph/BatteryMeterDrawableBase.java
@@ -347,6 +347,7 @@ public class BatteryMeterDrawableBase extends Drawable {
                 strokeWidth / 2.0f,
                 circleSize - strokeWidth / 2.0f + mPadding.left,
                 circleSize - strokeWidth / 2.0f);
+        mFrame.offset(-2.0f, 0);
 
         // set the battery charging color
         mBatteryPaint.setColor(batteryColorForLevel(level));
-- 
2.17.1

